modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -44,6 +45,7 @@ the registry. For example:
 . Verify that the `local_registry_images.yaml` file contains the Octavia images.
 For example:
 +
+[source,yaml]
 ----
 ...
```

Comment:
Our stylesheet seems to underline the ellipses here. Is that intended?

Reply:
> Our stylesheet seems to underline the ellipses here. Is that intended?

I am not sure. @jboxman, is this something that you've considered in the CSS changes?

Reply:
@maxwelldb @kalexand-rh that's part of the style as written; I've noticed that but cannot say why the author thought it made sense to do so. I can certainly remove it in a future update to the CSS.